pkgs/development/python-modules/remctl/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, buildPythonPackage, isPy27, remctl, pytestCheckHook, pytestrunner, typing }:
+
+buildPythonPackage {
+  inherit (remctl) pname version meta;
+  src = remctl.pythonsrc;
+  postPatch = ''
+    sed -i '/install_requires/d' setup.py
+  '';
+  buildInputs = [ pkgs.remctl ];
+  checkInputs = [ pytestCheckHook pytestrunner ];
```

Comment:
pytestrunner is deprecated. Can you try patching the dependency out and see if it still works?